.github/workflows/e2eMinVscodeVersion.yml

```
@@ -0,0 +1,95 @@
+name: End to End Tests for VS Code Minimum Supported Version
+on:
+  workflow_run:
+    workflows:
+      - Nightly Build Develop
```

We still want to test the minimum supported VSCode version before releases.

```suggestion
    workflows:
      - Nightly Build Develop
      - Test, Build, and Release
      - Create and Test Beta Release Branch
```